app/components/footer/feedback_component.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="feedback-bar">
   <section class="container">
     <h2 class="heading-s heading--box-blue">FEEDBACK</h2>
-    <p><a target="_blank" rel="noopener" href="https://forms.office.com/e/zVfV4SZeVn">
+    <p><a rel="noopener" href="/feedback">
```

Comment:
FYI this component has now been removed from the front end @StackedPancakez 

We have a separate ticket in the backlog to clear up the code